BridgeEmulator/HueEmulator3.py

```diff
@@ -518,6 +541,12 @@ def sendLightRequest(light, data):
                 elif key == "xy":
                     payload["5709"] = int(value[0] * 65535)
                     payload["5710"] = int(value[1] * 65535)
+            pprint(data)
+            if "hue" in data or "sat" in data:
```

Comment:
replace `if "hue" in data or "sat" in data:` with `elfi` and put it in same line with previews commands. This way is more efficient because this "if" will not be processed if you change any other values. If you put it i a new line it will be processed every time